products/issue_tracker/backend/models.py

```diff
@@ -0,0 +1,184 @@
+from django.db import models
+from django.utils import timezone
+import uuid
+
+
+class Issue(models.Model):
+    class Status(models.TextChoices):
```

Comment:
Probably ok for now but maybe we want the status to be a separate model. I'd imagine teams will want to be able to create their own stages